src/dialogs/more-info/controls/more-info-update.ts

```diff
@@ -64,7 +64,8 @@ class MoreInfoUpdate extends LitElement {
     try {
       const config = await getSupervisorUpdateConfig(this.hass);
 
-      if (type === "home_assistant") {
+      // for home assistant and OS updates
+      if (type.startsWith("home_assistant")) {
```

Comment:
I would make it explicit, as if we change something in the future this could easily break.
```suggestion
      if (["home_assistant", "home_assistant_os"].includes(type)) {
```

Reply:
Or make it a helper function that you can reuse, as it is used in 3 places